src/modules/jcms_rest/src/Plugin/Field/FieldTypeExtend/IntegerItem.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Drupal\jcms_rest\Plugin\Field\FieldTypeExtend;
+
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\Plugin\Field\FieldType\IntegerItem as IntegerItemExtend;
+
+/**
+ * {@inheritdoc}
+ */
+class IntegerItem extends IntegerItemExtend {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function generateSampleValue(FieldDefinitionInterface $field_definition) {
+    $values = parent::generateSampleValue($field_definition);
+    switch ($field_definition->getName()) {
+      // Random values must be unique.
+      case 'field_annual_report_year':
+        static $unique = [];
+        while (in_array($values['value'], $unique)) {
+          $values = parent::generateSampleValue($field_definition);
```

Comment:
pay attention to the `minmum` constraint https://github.com/elifesciences/api-raml/blob/67820ba2054346ab47f2a492b8e9f9755aad8c48/src/model/annual-report.v1.yaml#L8

Reply:
The parent method will consider the minimum value of `2012` and also the maximum which is set in the configuration.

See: https://github.com/elifesciences/journal-cms/blob/cc3d7eba87d2d8ea81f012c59f5371d318d4158b/sync/field.field.node.annual_report.field_annual_report_year.yml#L19-L22